src/portals/hope/features/claims/claim-details/ClaimCounterparty/InvoiceModal.tsx

```diff
@@ -0,0 +1,218 @@
+        claimType
+      }
+      bankgiroNumber
+      bankAccountNumber
+      paymentDueDate
+      counterparty {
+        id
+        reference
+        insuranceProvider {
+          displayName
+        }
+      }
+    }
+  }
+`
+
+export const InvoiceModal: React.FC<{
+  visible: boolean
+  onClose: () => void
+  invoice: RecoveryInvoice | null
+}> = ({ visible, onClose, invoice }) => {
+  const [updateRecoveryInvoiceContent] =
+    useUpdateRecoveryInvoiceContentMutation()
+
+  const content =
+    invoice === null
+      ? ''
+      : `
+        <div>
+          <p style="margin:0">Invoice Number:
+              <span style="color:${lightTheme.accent}">${
+          invoice.invoiceNumber
+        }</span>
+          </p>
+          <p style="margin:0">Our reference: 
+              <span style="color:${lightTheme.accent}">${
+          invoice.memberReference
+        }</span>
+          </p>
+          <p style="margin:0">Your reference: 
+              <span style="color:${lightTheme.accent}">${
+          invoice.counterparty.reference
+        }</span>
+          </p>
+        </div>
+        
+        <div style="margin-top:1rem">
+          <p style="margin:0">Claim case happened on 
+            <span style="color:${lightTheme.accent}">${
+          invoice.dateOfOccurrence
+        }</span>
+          </p>
+          <p style="margin:0">Paying party: 
+            <span style="color:${lightTheme.accent}">${
+          invoice.counterparty.insuranceProvider?.displayName
+        }</span>
+          </p>
+        </div>
+        
+        <div style="margin-top:1rem">
+          <p style="margin:0">We ask you to process our claim of 
+            <span style="color:${lightTheme.accent}">${parseFloat(
+          invoice.amount.amount,
+        )}</span> SEK regarding our expenses in the incident.
+          </p>
+        </div>
+
+        <div style="margin-top:1rem">
+          ${invoice.recoveryData.map(
+            (data) => `
+            <p>
+              ${convertEnumToTitle(data.claimType)}: 
+              <span style="color:${lightTheme.accent}">${parseFloat(
+              data.amount.amount,
+            )} SEK</span>
+            </p>`,
+          )}
+        </div>
+
+        <div style="margin-top:1rem">
+          <p style="margin:0">Total: 
+            <span style="color:${lightTheme.accent}">
+                ${invoice.recoveryData.reduce(
+                  (acc, data) => acc + parseFloat(data.amount.amount),
+                  0,
+                )} SEK
+            </span>
+          </p>
+        </div>
+
+        <div style="margin-top:1rem">
+          <p style="margin:0">
+            The amount can be deposited in our Bank Giro: 
+            <span style="color:${lightTheme.accent}">${
+          invoice.bankgiroNumber
+        }</span> 
+            or Bank Account: 
+            <span style="color:${lightTheme.accent}">${
+          invoice.bankAccountNumber
+        }</span>
+          </p>
```

Comment:
I think it'd be easier to read these if they were on separate lines
```suggestion
          <p style="margin:0">
            The amount can be deposited in our:</p>
         <p style="margin:0">
            Bank Giro: 
            <span style="color:${lightTheme.accent}">${
          invoice.bankgiroNumber
        }</p> 
        <p style="margin:0">
            Bank Account: 
            <span style="color:${lightTheme.accent}">${
          invoice.bankAccountNumber
        }</span>
          </p>
```